frontend/packages/db-structure/src/parser/prisma/parser.ts

```diff
@@ -229,4 +273,143 @@ function normalizeConstraintName(constraint: string): ForeignKeyConstraint {
   }
 }
 
+/**
+ * Creates the name for a many-to-many join table
+ */
+function createManyToManyJoinTableName(model1: string, model2: string): string {
+  return `_${model1}To${model2}`;
+}
+
+/**
+ * Creates a join table for a many-to-many relationship
+ */
+function createManyToManyJoinTable(joinTableName: string): Table {
+  return {
+    name: joinTableName,
+    columns: {
+      A: {
+        name: 'A',
+        type: 'serial',
```

Comment:
There are some specifications that are not supported by this PoC, including `type`. Please investigate and implement them as well.

https://www.prisma.io/docs/orm/prisma-schema/data-model/relations/many-to-many-relations#rules-for-defining-an-implicit-m-n-relation